checks.d/win32_event_log.py

```diff
@@ -150,13 +151,14 @@ def _convert_event_types(self, types):
 
 
 class LogEvent(object):
-    def __init__(self, ev, api_key, hostname, tags, notify_list):
+    def __init__(self, ev, api_key, hostname, tags, notify_list, tag_event_id):
         self.event = ev
         self.api_key = api_key
         self.hostname = hostname
         self.tags = tags
```

Comment:
I might be missing something, but why not use something like `self.tags = self._tags(tags, ev.EventCode) if tag_event_id else tags` ?
This way you don't need to have a `self.tag_event_id = tag_event_id`.